Package.swift

```diff
@@ -6,7 +6,8 @@ import PackageDescription
 let package = Package(
     name: "SpruceIDWalletSdk",
     platforms: [
-        .iOS(.v14)
+        .iOS(.v13),
+        .macOS(.v10_15)
```

Comment:
Rust would need to produce an artifact for macOS, perhaps this should be removed until it does.